themes/bootstrap3/templates/RecordDriver/DefaultRecord/data-publicationDetails.phtml

```diff
@@ -4,7 +4,11 @@
     <span property="location"><?=$this->escapeHtml($pubPlace)?></span>
   <?php endif; ?>
   <?php $pubName = $field->getName(); if (!empty($pubName)): ?>
-    <span property="name"><?=$this->escapeHtml($pubName)?></span>
+    <span property="name">
```

Comment:
I'm not a microdata expert, but based on the [RDFa Lite](https://www.w3.org/TR/rdfa-lite/#vocab-typeof-and-property) documentation, I think you can put the `property="name"` attribute directly onto the `<a>` tag instead of wrapping it in a `<span>` in order to simplify the markup. Would you agree?

Reply:
I do agree. This is done now